Dockerfile.dapper

```
@@ -79,14 +79,15 @@ RUN if [[ "${ARCH}" == "amd64" ]]; then \
 ENV YQ_VERSION v4.41.1
 RUN wget -q https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_${ARCH}.tar.gz -O - | tar xz && mv yq_linux_${ARCH} /usr/bin/yq
 
-ENV DAPPER_ENV REPO TAG CI DRONE_BUILD_NUMBER DRONE_BUILD_EVENT DRONE_TAG DRONE_COMMIT_BEFORE \
+ENV DAPPER_ENV REPO TAG CI \
+    PREV_COMMIT_PR_SHA PREV_COMMIT_PUSH_SHA GITHUB_EVENT_NAME GITHUB_RUN_NUMBER GITHUB_REF_TYPE GITHUB_REF_NAME \
     REGISTRY_ENDPOINT REGISTRY_USERNAME REGISTRY_PASSWORD \
     V2PROV_TEST_DIST V2PROV_TEST_RUN_REGEX KDM_TEST_K8S_MINOR DEBUG
 ENV DAPPER_SOURCE /go/src/github.com/rancher/kontainer-driver-metadata
 ENV DAPPER_DOCKER_SOCKET true
 ARG CI
-ARG DRONE_BUILD_NUMBER
-ENV DAPPER_RUN_ARGS "--privileged --label CI=${CI} --label DRONE_BUILD_NUMBER=${DRONE_BUILD_NUMBER}"
+ARG GITHUB_RUN_NUMBER
+ENV DAPPER_RUN_ARGS "--privileged --label CI=${CI} --label DRONE_BUILD_NUMBER=${GITHUB_RUN_NUMBER}"
```

```suggestion
ENV DAPPER_RUN_ARGS "--privileged --label CI=${CI} --label GITHUB_RUN_NUMBER=${GITHUB_RUN_NUMBER}"
```
Do we still need this label to say `DRONE_BUILD_NUMBER` for any reason, or can we change it?

---

its being used in rancher/rancher repo for some sort of cleanup.. so will need to be fixed there first..
https://github.com/search?q=repo%3Arancher%2Francher%20DRONE_BUILD_NUMBER&type=code
Since rancher repo is being handled by a different team, did not want to mess around with it, without knowing the full extent of its impact on rancher repo as well. So for the time being left it as it is.